apps/site/src/lib/components/Button/Button.tsx

```diff
@@ -2,26 +2,30 @@ import type { ComponentProps } from "react";
 import Link from "next/link";
 import styles from "./Button.module.css";
 import clsx from "clsx";
+import { disableInstantTransitions } from "framer-motion";
 
 interface ButtonProps {
 	text: string;
 	className?: string;
 	href?: ComponentProps<typeof Link>["href"];
 	isLightVersion?: boolean;
+	usePrefetch?: boolean;
 }
 
 const Button: React.FC<ButtonProps> = ({
 	text,
 	href,
 	className,
 	isLightVersion,
+	usePrefetch
```

Comment:
Note [out of scope]: we could extend the `InternalLinkProps` and pass `...rest` so we don't have to individually specify link props we need.